test/users/signs_up.test.js

```diff
@@ -0,0 +1,51 @@
+const supertest = require('supertest');
+
+const app = require('../../app');
+const db = require('../../app/models/index');
+const { userDataCorrectly, wrongPassword } = require('../factory/users');
+
+const logger = require('../../app/logger');
+
+describe('Users - Sign Up', () => {
+  it('Response status 200 - User created correctly', async done => {
+    const response = await supertest(app)
+      .post('/users')
+      .send(userDataCorrectly())
+      .set('Accept', 'application/json');
+    expect(response.status).toBe(200);
```

Comment:
In cases of success, it would be good to check, apart from the status, that the response we expect is also arriving.

Reply:
In this case that the object is one of  type user (serializers)?